### ISSUE TITLE
Where are the pre-trained models come from?

### ISSUE THREAD

**Issue by SayHelloToWorld** [BichenWuUCB/squeezeDet#82]
Recently, I want to write a neural network based on your code. So, can I get the source code about how you train your squeezenet classification network? Thanks a lot!

**Comment by luhaijun** [comment]
I am now training on Kitty data to remove the pre-trained model, but the accuracy rate is about 0.1%,but i cant find why
